src/Components/ToDo/index.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+
+import Typography from '@material-ui/core/Typography';
+import Container from '@material-ui/core/Container';
+
+import store from '../../Components/ToDo/SagaStore/store';
+
+import AddToDo from './AddToDo';
+import ToDoList from './ToDoList';
+// import ResetButton from './resetButton';
+
+const ToDo = () => (
+  <Provider store={store}>
+    <Container maxWidth="md">
+      <Typography style={{ padding: '20px', color: 'blue', margin: '20px' }} variant="h5">
```

Comment:
Don't use inline styles
try using makeStyles [https://material-ui.com/styles/advanced/#makestyles](https://material-ui.com/styles/advanced/#makestyles)

Reply:
Correct, Some of issues I'm facing when trying to use makeStyles.
I'll update here.